.travis.yml

```diff
@@ -1,6 +1,16 @@
+sudo: required
+
 language: ruby
 rvm:
   - 2.0
```

Comment:
Not sure, if those versions are no longer supported/used on MIQ it would be good to trim them out.